src/utils/scales-utils.js

```diff
@@ -177,19 +186,19 @@ export function getScaleFnFromScaleObject(scaleObject) {
 /**
  * Get the domain from the array of data.
  * @param {Array} allData All data.
- * @param {string} attr Property name.
+ * @param {function} accessor - accessor for main value.
+ * @param {function} accessor0 - accessor for the naught value.
```

Comment:
Something about these names bother me, especially `accessor0`. Maybe call the params, `mainAccessor`, `zeroAccessor`

Haven't read carefully, but why do we have an accessor for zero as opposed to just supplying a value? Assuming this is a single value?

+1 for dropping `attr`, by the way.

Reply:
So to clarify you are suggesting a naught accessor like zeroX and zeroAngle?

Reply:
> To to clarify you are suggesting a naught accessor like zeroX and zeroAngle?

To be honest, I am not completely sure what I am proposing as I don't fully understand what the zero accessors are doing. Mainly reacting to the name which looks unusual and not immediately obvious to a new comer.

First of all, I am asking if `accessor0` needs to be an accessor, e.g.
* Is `accessor0` called on every item, then yes,
* but if `accessor0` is called just once, to get a single "zero" value as reference point for the series, could we just supply the zero value directly rather than ask for another accessor?

If only once, then yes, (value) props could be called ` zeroX` and `zeroAngle`

If it needs to be an accessor then my case is weaker. It could still be worth at least looking at naming options, like `getZeroX` and `getZeroAngle`.

Reply:
The zero accessor tend to come up in a few particular types of cases. For instance if you have rectangle expressed through RectSeries, you'd express the bottom of the rectangle through getY0 and the top through getY. Similarly, if you were working with an ArcSeries, getAngle0 would express the start angle of the wedge and getAngle would express the end angle. Each of these refers to values found on the data eg {angle0: Pi, angle: Pi / 2} or {y0: 2, y: 4}, so it'd be a pretty big change to alter this naming convention too.

Reply:
Got it. Starting fresh, I would personally favor changing the expected data fields to `startAngle` etc. But agree that staying consistent now makes the most sense.

Perhaps something to consider in the API audit @jckr is planning.